packages/web/src/common/design/components/Switch.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+import {
+    LabelSwitch,
+    InputSwitch,
+    SpanSwitch
+} from "../../../modules/core/styles/switchStyle"
+
+/*
+    To do:
+    That goddamn interfaace that I dont know how to do it to mention the props (I think)
+*/
+const Switch = props => {
```

Comment:
Enforce correct FunctionComponent type
```suggestion
const Switch: React.FC = props => {
```

Reply:
done